ui-v2/app/adapters/application.js

```diff
@@ -62,7 +94,7 @@ export default Adapter.extend({
     }
     // TODO: we could use a URL here? They are unique AND useful
     // but probably slower to create?
-    return JSON.stringify([dc, slug]);
+    return hash([dc, slug]);
```

Comment:
Do you do something _other_ than `JSON.stringify` in other places?

Reply:
In the unit tests:

https://github.com/hashicorp/consul/blob/c03b141a41812e994cae2361adbcc0d6cf977afc/ui-v2/tests/unit/adapters/application-test.js#L21-L35

Reply:
Oh! Nice 👌